src/raspberry_pi_ui/rokku.py

```diff
@@ -220,12 +220,46 @@ def on_soundAlarmButton_clicked(self, widget):
     """
 
     def on_recordButton_clicked(self, widget):
-        # add code to check if clip is still recording and set self.recording accordingly
-
+        # variable to hold link as a string
+        yt_link = None
```

Comment:
Consider make `yt_link` an attribute of the class (i.e. `self.yt_link`), because it might be used later by other methods to access the link.

Reply:
Done, and I made two, 1 for the playlist and 1 for the live stream.